server.js

```diff
@@ -125,14 +132,18 @@ io.on('connection', (socket) => {
 
   // PHASE IV
   // User Disconnects:
+  rooms = []
   socket.on('disconnect', () => {
     // console.log(socket.id + ' DISCONNECTED')
     var user_room_id = socket.id;
     var room = io.sockets.adapter.rooms[user_room_id];
+    console.log("Socket that dc'd: " + user_room_id);
+    console.log("Role: " + socket.role);
     if (room) {
       // room exists, either admin or user left in room, send disconnect
-      socket.broadcast.to(user_room_id).emit('user disconnect', user_room_id);
-    } else {
+      // socket.broadcast.to(user_room_id).emit('user disconnect', user_room_id);
```

Comment:
Why comment this out? If this is what you intended, could you reform the if statement to not have an empty clause?